.travis.yml

```diff
@@ -94,7 +94,7 @@ jobs:
     - name: "Release"
       stage: release
       name: "Push to Nexus"
-      if: fork=false AND commit_message ~= /\[trigger release\]/ AND branch ~= /^(master|SP\/.+|HF\/.+)$/
+      if: fork=false AND commit_message =~ /\[release\]/ AND branch ~= /^(master|SP\/.+|HF\/.+)$/
```

Comment:
Why do we have commit_message **_=~_** /\[release\] and  branch **_~=_** /^(master|SP\/.+|HF\/.+)$/ ? What is the correct form =~ or ~= ?